lib/logflare_web/live/backends/components/backend_form.heex

```diff
@@ -214,6 +212,10 @@
               The port number for the ClickHouse server.
             </small>
           </div>
+          <div class="form-group">
+            <%= label(f_config, :port, "Connection pool size") %>
+            <%= text_input(f_config, :pool_size, class: "form-control", default: 1) %>
```

Comment:
does this actually propagate to the ch pool? seems like we're hardcoding it to 3